### DISCUSSION TITLE
Flächen zusammenführen (merge)

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#172] [General]
Das Zusammenführen vom Flächen ist eine große Erleichterung, wenn große Flächen erfasst werden sollen --> so zeichnet man zuerst die Teilflächen und führt diese zum Schluss zusammen, damit sollten auch innerhalb der zusammenzuführenden Teilflächen die Überlappungen bereinigt werden --> so mein Gedankengang.
Das Verhalten dieses Tools im Editieren verhält sich im webGIS5 (bei uns produktiv) anders als im webGIS6 (wird noch getestet):
|webgis 5.24.1801| webgis 6.24.601|
|--------------------|--------------------|
|![image](https://github.com/gis-eni/webgis-forum/assets/1842500/81baa875-8274-4426-85a4-500544023303)|![image](https://github.com/gis-eni/webgis-forum/assets/1842500/0055cc68-a9b0-4ef1-a733-07afc66ed643)|

Wobei die webGIS 5 Zusammenführung mMn nicht das erwartete Ergebnis bringt. Wird das gefixt, oder sollen wir schon auf webGIS 6 umsteigen?

**Comment by gis-eni** [comment]
Kommt nicht mehr im WebGIS 5
